### ISSUE TITLE
page with 3 systems recognized as a big one - impossible to separate

### ISSUE THREAD

**Issue by Bacchushlg** [Audiveris/audiveris#656]
![I'm a train 4](https://github.com/Audiveris/audiveris/assets/33522630/80faffe6-4507-40f6-9203-d4cfd92f2639)

This page with 3 systems is recognized as a single big system - I find no way to correct this.

**Comment by hbitteur** [comment]
If I remove the vertical line on the left margin of the image (and the horizontal line on top as well), then everything works fine.

I did this with an image editor (XnView).
So I'm putting this case under the "Regions" label by which the user should be able to blacklist certain regions, not necessarily rectangular ones.

**Comment by Bacchushlg** [comment]
Ah, ok.
But maybe this may appear again... Does it make sense to provide a function as conterpart to "merge systems" that would allow to solve the problem directly. The solution that you found might not be logical to a normal user...

**Comment by hbitteur** [comment]
Yes, something like a "split system at this vertical location" action available in the right-click popup menu.

**Comment by phdm** [comment]
This is exactly the same problem as #652.
